wurst/systems/units/hostiles/trolls/Scout.wurst

```diff
@@ -21,6 +21,12 @@ public class Scout extends Troll
     override function getTypeId() returns int
         return unitTypeId
 
+    override function postCreate()
+        super.postCreate()
+
+        getUnit().addAbility(SPELL_PING_ENEMY)
+        //getUnit().addAbility(SPELL_TRAP)
+        //getUnit().addAbility(SPELL_SPIKED_TRAP)  
```

Comment:
Let's put these behind a mode

Reply:
SPELL_PING_ENEMY is the unsub scout radar wurstified, ping elk in green now, the rest was for testing purpose, will remove it.